crates/sidevm/host/src/web_api.rs

```diff
@@ -284,11 +291,26 @@ async fn info(app: &State<App>) -> String {
         "running": sidevm_host_runtime::vm_count(),
         "deployed": inner.instances.len(),
         "ids": inner.instances.keys().cloned().collect::<Vec<_>>(),
-    }).to_string()
+    })
+    .to_string()
 }
 
 pub async fn serve(args: Args) -> anyhow::Result<()> {
-    let (run, spawner) = sidevm::service(args.workers);
+    let (tx, mut rx) = tokio::sync::mpsc::channel(10);
+    let (run, spawner) = sidevm::service(args.workers, tx);
+    tokio::spawn(async move {
+        while let Some((id, message)) = rx.recv().await {
+            let OutgoingRequest::Query {
+                contract_id,
+                payload,
+                reply_tx,
+            } = message;
+            let vmid = ShortId(id);
+            let dest = ShortId(contract_id);
+            info!(%vmid, "Outgoing message to {dest} payload: {payload:?}");
+            _ = reply_tx.send(Vec::new());
```

Comment:
Why to send an empty vec?

Reply:
This is the standalone sidevm runner, which doesn't have pink runtime. We might add mocking API in the future. Currently, maybe sending an error is better, but anyway, it just doesn't support for testing with pink ATM.

Reply:
Good to know